### ISSUE TITLE
Jackett and Radarr cooperation not working

### ISSUE THREAD

**Issue by MaddionMax** [Radarr/Radarr#2535]
The main problem:

Jackett setup done, working with trackert. If I search from Jackett a movie from an indexer, found the movie perfectly. Also setup Radarr and added Jackett indexers and wrote in Jackett's manual.
Now Radarr not find the same movie.

If I set in the profile english language, than find the hungarian version and mark as it's not what I seeking. If I set the hungrian language in profile, than not appear the movie.

What is the problem?

**Comment by MaddionMax** [comment]
Tested with all other languages, the result is the same. If the profile main language not hungarian, Radarr find the hungarian. If the hungarian set, not found any result,

**Comment by MaddionMax** [comment]
Uploaded the images to an album.  https://unsee.cc/a36fabbe/

Problem identified:

If I change the language to english on Radarr profile, than Radarr find the movie but mark as it's not hungarian what I want. If I change back the profile to hungarian, Radarr not find anything.

I also checked everything directly from Jackett, it's worked perfectly.

**Comment by galli-leo** [comment]
Did you do a movie refresh after changing the profile to hungarian? Also can you post a picture of your titles tab with the profile set to hungarian?

**Comment by MaddionMax** [comment]
Of course I done it.

https://unsee.cc/a1ba222a/   The 3rd image is the hungarian profile

**Comment by MaddionMax** [comment]
Uploaded it

**Comment by galli-leo** [comment]
Fixed on latest nightly.

**Comment by MaddionMax** [comment]
Changed to nightly, still not working.

Screenshot uploaded to the folder. Hungarian language on profile, not find anything on indexer.

**Comment by galli-leo** [comment]
Did you do a movie refresh?

**Comment by MaddionMax** [comment]
Of course

**Comment by MaddionMax** [comment]
Deleted the test movie what I added and added it again...now started to work!
Thanks galli-leo!

**Comment by MaddionMax** [comment]
Sorry to bother you again, but again not working.

Added yesterday some movies and worked perfectly.
Today I added one movie (Schindler's List) and not able to find it on indexer.

Enclosed images here: https://unsee.cc/05b58394/

The title is goond, indexer on Jackett is working (as your see Jackett found several copies on the indexer) and after several movie refresh, Radarr not found any copy. Also added screenshot about my profile, but it's same as yesterday.
